src/docker/Dockerfile

```diff
@@ -136,6 +136,14 @@ RUN dnf -y install podman-${PODMAN_VERSION} \
   && rm -rf /var/cache/yum/ \
   && rm -rf /var/tmp/* /tmp/* /var/tmp/.???* /tmp/.???*
 
+# Install Skopeo. Get available versions by running the command `dnf list skopeo`
+ARG SKOPEO_VERSION=“1.2.0”
```

Comment:
Change smart quotes to regular double quotes

Reply:
done